packages/typespec-azure-portal-core/package.json

```
@@ -0,0 +1,53 @@
+{
+  "name": "@azure-tools/typespec-azure-portal-core",
+  "version": "1.0.0",
```

```suggestion
  "version": "0.39.0",
```

(or match the other azure packages version at the time before merging)

---

might be good to add that package to `.chornus.yaml` version policy